test/system/030-run.bats

```diff
@@ -1021,4 +1021,35 @@ EOF
     run_podman run --net=host --cgroupns=host --rm $IMAGE sh -c "grep ' / /sys/fs/cgroup ' /proc/self/mountinfo | tail -n 1 | grep '/sys/fs/cgroup ro'"
 }
 
+@test "podman run - rootfs with idmapped mounts" {
+    skip_if_rootless "idmapped mounts work only with root for now"
+
+    skip_if_remote "userns=auto is set on the server"
+
+    egrep -q "^containers:" /etc/subuid || skip "no IDs allocated for user 'containers'"
+
+    run_podman image mount $IMAGE
+    src="$output"
+
+    # check if the underlying file system supports idmapped mounts
+    mkdir -p $PODMAN_TMPDIR/idmap-check
+    mount --bind -o X-mount.idmap='0:1:1' $PODMAN_TMPDIR $PODMAN_TMPDIR || skip "idmapped mounts not supported"
```

Comment:
Suggestion, to avoid leaving behind a podman-mount:
```diff
-    mount --bind -o X-mount.idmap='0:1:1' $PODMAN_TMPDIR $PODMAN_TMPDIR || skip "idmapped mounts not supported"
+    echo "$_LOG_PROMPT mount --bind -o X-mount.idmap=0:1:1 ..."
+    run mount --bind -o X-mount.idmap='0:1:1' $PODMAN_TMPDIR $PODMAN_TMPDIR
+    echo "$output"
+    if [[ $status -ne 0 ]]; then
+        run_podman image unmount $IMAGE
+        skip "idmapped mounts not supported"
+    fi
+
```

Reply:
It is not a "podman mount", it is a raw mount command.  How can it leak (well assuming the script is not killed in the middle) if the next command is an "umount"?

Reply:
I've moved the `'podman mount` later, does the new version look fine to you?

Reply:
Yes! That's a much better solution, thank you!

Test still fails on my laptop though. (Same EINVAL)

Reply:
does the `run mount --bind -o X-mount.idmap='0:1:1' $PODMAN_TMPDIR $PODMAN_TMPDIR` command work fine on your laptop?

Reply:
ah, mount will silently ignore the option if not supported :/

I need to figure out a different way to check the feature

Reply:
I've not found any way to test it, so I am reusing the podman command and check its error message